pom.xml

```diff
@@ -904,8 +904,37 @@
                     <configuration>
                         <fail>false</fail>
                         <rules>
+                            <banCircularDependencies />
+                            <banDependencyManagementScope />
+                            <banDuplicateClasses>
+                                <dependencies>
+                                    <dependency>
+                                        <groupId>org.checkerframework</groupId>
+                                        <artifactId>dataflow-errorprone</artifactId>
+                                        <!-- This package is contained in
+                                        Checker Framework's `checker-qual` and
+                                        `dataflow-errorprone` modules. Error
+                                        Prone requires the latter, while Guava
+                                        pulls in the former. Since the package
+                                        contains mostly annotations, this is
+                                        not expected to cause trouble in
+                                        practice. -->
+                                        <ignoreClasses>
+                                            <ignoreClass>org.checkerframework.dataflow.qual.*</ignoreClass>
+                                        </ignoreClasses>
+                                    </dependency>
+                                </dependencies>
+                                <findAllDuplicates>true</findAllDuplicates>
+                            </banDuplicateClasses>
                             <banDuplicatePomDependencyVersions />
+                            <!-- XXX: Enable this rule once it no longer
+                            "downloads the internet". See
+                            https://issues.apache.org/jira/browse/MENFORCER-467.
+                            <banDynamicVersions />-->
                             <dependencyConvergence />
+                            <enforceBytecodeVersion>
```

Comment:
Another [link](https://www.mojohaus.org/extra-enforcer-rules/enforceBytecodeVersion.html) for easy clicking.